src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="iMac Image"
+      />
+      <a
+        class="product-card__name"
+        href="#"
+      >
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
+      <p class="product-card__code">Product code: 195434</p>
```

Comment:
don't forget about indents. It will be more readable
```suggestion
      <img
        class="product-card__image"
        src="images/imac.jpeg"
        alt="iMac Image"
      />
      
      <a
        class="product-card__name"
        href="#"
      >
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </a>
      
      <p class="product-card__code">Product code: 195434</p>
```